pkgs/by-name/qu/quarkus/package.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchurl
+, writeScript
+, bash
+, openjdk
+}:
+
+stdenv.mkDerivation (finalAttrs: rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

One of the `finalAttrs`'s goals was to remove `rec` in `mkDerivation`:
https://nixos.org/manual/nixpkgs/stable/#mkderivation-recursive-attributes

---

yep, brain fart, sorry